src/rust-2018/macros/macro-changes.md

```diff
@@ -78,3 +80,192 @@ struct Bar;
 
 This only works for macros defined in external crates.
 For macros defined locally, `#[macro_use] mod foo;` is still required, as it was in Rust 2015.
+
+### Local helper macros
```

Comment:
It may be useful for authors of crates using _foreign_ helper macros to also have some examples. Those examples may almost look exactly the local examples below so it may be even better to rename this sections `helper macros` with examples using a mix of both `foreign-crate::` and `$crate::` alike.

Reply:
Would it be possible to merge this PR first, and then address this in a new PR?